### PR TITLE
bump timeout for goreleaser to 60 minutes.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Summary
Looks like the releaser is right at the hairy edge, so bump it.
https://github.com/sigstore/policy-controller/actions/runs/4038223564/jobs/6942272664

Looks like it's been hovering there, so just bumping to 60. Previous runs:
https://github.com/sigstore/policy-controller/actions/runs/3670391147/jobs/6204895886

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]
let's see if that works

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
         uses: goreleaser/goreleaser-action@8f67e590f2d095516493f017008adc464e63adb1 # v4.1.0
         with:
           version: latest
-          args: release --rm-dist
+          args: release --rm-dist --timeout 60m
```
Comment (cpanato) [inline .github/workflows/release.yaml:60]: i would add 120m just to be sure :)

Comment (hectorj2f) [reply .github/workflows/release.yaml:60]: we need to add a COSIGN_YES env variable. I'll do it.

Comment (cpanato) [reply .github/workflows/release.yaml:60]: https://github.com/sigstore/policy-controller/pull/540

Comment (hectorj2f) [reply .github/workflows/release.yaml:60]: 👍🏻

**Pull request merged**